src/extensions/replay/config.ts

```diff
@@ -68,7 +68,7 @@ const removeAuthorizationHeader = (data: CapturedNetworkRequest): CapturedNetwor
     return data
 }
 
-const POSTHOG_PATHS_TO_IGNORE = ['/s/', '/e/', '/i/vo/e/']
+const POSTHOG_PATHS_TO_IGNORE = ['/s/', '/e/', '/i/v0/e/', '/i/vo/e/', '/decide/', '/static/recorder-v2.js']
```

Comment:
was `vo` a typo or a valid one in the past?

Reply:
Lets also ignore the recorder? URL https://eu-assets.i.posthog.com/static/recorder-v2.js?v=1.110.0

Reply:
totally a typo...

let's keep it to just ingestion paths for now... others can be useful when viewing our own site